backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json

```diff
@@ -0,0 +1,14 @@
+{
+  "PlatformSettings": {
+    "ApiAccessManagementEndpoint": "https://platform.at21.altinn.cloud/accessmanagement/api/v1/",
+    "ApiAuthenticationEndpoint": "https://platform.at21.altinn.cloud/authentication/api/v1/",
+    "ApiProfileEndpoint": "https://platform.at21.altinn.cloud/profile/api/v1/",
+    "ApiRegisterEndpoint": "https://platform.at21.altinn.cloud/register/api/v1/",
+    "ApiResourceRegistryEndpoint": "https://platform.at21.altinn.cloud/resourceregistry/api/v1/",
+    "OpenIdWellKnownEndpoint": "https://platform.at21.altinn.cloud/authentication/api/v1/openid/"
+  },
+  "GeneralSettings": {
+    "FrontendBaseUrl": "https://am.ui.at21.altinn.cloud/",
+    "Hostname": "at21.altinn.cloud"
+  }
+}
```

Comment:
We also need cache config, client settings, keyvaultsettings

Reply:
I guess we need to save the clientsettings and keyvaultsettings as environment secret

Reply:
cache config will for now just use the defaults in appsettings.json. 
No need to introduce environment specific values until we actually have different values for different environments.

yes, the keyvault and client settings should come from environment secrets